### ISSUE TITLE
`impt log stream` fails with custom / Private Cloud endpoint behind proxy server

### ISSUE THREAD

**Issue by deldrid1** [EatonGMBD/imp-central-impt#5]
Other `impt` commands like `impt build run` are working fine with the custom endpoint

```bash
$ impt log stream -d 30000c2a690e1d03 -z debug
Doing the request with options:
{
  "url": "https://[hidden]/v5/devices/30000c2a690e1d03",
  "method": "GET",
  "headers": {
    "Content-type": "application/vnd.api+json",
    "Authorization": "[hidden]"
  },
  "json": true,
  "qs": null,
  "body": null,
  "qsStringifyOptions": {
    "arrayFormat": "repeat"
  },
  "forever": true
}
Error: Request error: Error: getaddrinfo ENOTFOUND [hidden] [hidden]:443
    at Function._getError (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/imp-central-api/lib/util/HttpHelper.js:110:20)
    at Request [as _callback] (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/imp-central-api/lib/util/HttpHelper.js:159:40)
    at self.callback (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/request/request.js:185:22)
    at Request.emit (events.js:189:13)
    at Request.onRequestError (/Users/e0088611/.nvm/versions/node/v10.15.3/lib/node_modules/imp-central-impt/node_modules/request/request.js:881:8)
    at ClientRequest.emit (events.js:189:13)
    at TLSSocket.socketErrorListener (_http_client.js:392:9)
    at TLSSocket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
Error: Request error: Error: getaddrinfo ENOTFOUND api.imp.[hidden].com api.imp.[hidden].com:443.
IMPT COMMAND FAILS
```

**Comment by alexey-nbl** [comment]
@deldrid1 
We cannot reproduce the issue. Pls see the comment in the EMCB-109.

**Comment by deldrid1** [comment]
EMCB-109 Updated - I left out a critical detail... This is only the case when behind a proxy server

**Comment by alexey-nbl** [comment]
@deldrid1 it duplicates https://github.com/EatonGMBD/imp-central-impt/issues/4 in this case

**Comment by mm-gmbd** [comment]
Fixed with #13
